src/frontends/pytorch/src/op/rsqrt.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_rsqrt(NodeContext& context) {
+    auto data = context.get_input(0);
+    auto input_shape = context.mark_node(std::make_shared<opset8::ShapeOf>(data));
+    auto one_const = context.mark_node(opset8::Constant::create(element::f32, Shape({}), {1}));
+    auto sqrt_data = context.mark_node(std::make_shared<opset8::Sqrt>(data));
+    auto divisor = context.mark_node(std::make_shared<opset8::Broadcast>(one_const, input_shape));
```

Comment:
I don't think explicit broadcast is needed here. `Divide` should auto broadcast just as any other elementwise op.

Reply:
removed, thanks